'

```diff
@@ -0,0 +1,24 @@
+# This is a combination of 2 commits.
```

Comment:
this file was probably added (and also created) by some mistake; it should be removed from PR?